recipe/meta.yaml

```diff
@@ -1,17 +1,21 @@
-{% set name = "libfuse" %}
 {% set version = "3.10.0" %}
 
 package:
-  name: {{ name }}
+  name: libfuse
   version: {{ version }}
 
 source:
-  url: https://github.com/{{ name }}/{{ name }}/releases/download/fuse-{{ version }}/fuse-{{ version }}.tar.xz
+  url: https://github.com/libfuse/libfuse/releases/download/fuse-{{ version }}/fuse-{{ version }}.tar.xz
   sha256: 26517954567f237a7dbcb532755ba0d2c77575c5d90db7566b6e40ec05b0a039
+  patches:
+    - 0001-Install-fusermount-init-script-into-sysconfdir.patch
+    - 0002-Enable-building-utils-on-CentOS-6.patch
 
 build:
-  number: 0
+  number: 1
   skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage('libfuse', max_pin='x.x') }}
```

Comment:
```suggestion
    # Seems to be quite stable between minor versions
    # https://abi-laboratory.pro/?view=timeline&l=libfuse
    - {{ pin_subpackage('libfuse', max_pin='x') }}
```

Reply:
Right, makes sense. I think I copy-pasted from another recipe and forgot to adjust it. Thanks.